src/main/java/com/denizenscript/ddiscordbot/DenizenDiscordBot.java

```diff
@@ -26,6 +27,7 @@
     public static Warning oldMessageContexts = new SlowWarning("dDiscordBot contexts relating to message data are now provided by DiscordMessageTag.");
     public static Warning oldMessageCommand = new SlowWarning("dDiscordBot's 'discord message' sub-command has been moved to a base 'discordmessage' command.");
     public static Warning oldConnectCommand = new FutureWarning("dDiscordBot's 'discord connect' sub-command has been moved to a base 'discordconnect' command.");
+    public static Warning globalSlashCommand = new SlowWarning("Registering a slash command globally may take up to an hour.");

```

Comment:
What part of this message constitutes a deprecation warning?

Reply:
It was originally just a warning for global slash commands but I've meant to remove it. I want to hear your opinion on it though first because I think something like this could be useful for unintended mistakes.

Reply:
It shouldn't be going through the deprecation warning system. If anything a `Debug.log` or `Debug.echoDebug`